Makefile

```diff
@@ -6,6 +6,10 @@ UNAME_S := $(shell uname -s)
 GIT_DATE := $(firstword $(shell git --no-pager show --date=short --format="%ai" --name-only))
 GIT_VERSION := $(shell git describe --abbrev=0 --tags)
 
+PREFIX=/usr/local
```

Comment:
Allow override of PREFIX, BINDIR, DATADIR and also support installation to arbitrary DESTDIR which is required by distro packaging.